ggu/main.c

```diff
@@ -880,13 +880,14 @@ run_ggu(struct net_config *net_conf,
  * Return an error if trying to allocate the second instance.
  */
 struct ggu_config *
-alloc_ggu_conf(void)
+alloc_ggu_conf(unsigned int lcore)
```

Comment:
Given that `alloc_ggu_conf()` now receives the lcore, set field `lcore_id` before returning and drop line `ggu_conf.lcore_id = lcore` from `lua/ggu.lua`.